test/userslist.test.js

```diff
@@ -69,35 +74,36 @@ describe('GET /users', () => {
 
   describe('Test token validation', () => {
     test('Create user, and ask users list with correct token', () =>
```

Comment:
Ill approve this, but keep in mind this is the bare minimum you could test about the core functionality (the missing params are very well tested).
For example, what happens if there are more than 10 users in the database?. Does it bring all of them? Does it bring only 10?
Just keep in mind for next PRs that more borderline cases like that one should be thought about and tested.